pkg/resource/sensitive.go

```diff
@@ -31,11 +31,18 @@ import (
 )
 
 const (
-	errCannotExpandWildcards = "cannot expand wildcards"
-
+	errCannotExpandWildcards          = "cannot expand wildcards"
 	errFmtCannotGetStringForFieldPath = "cannot not get a string for fieldpath %q"
 )
 
+const (
+	// PrefixAttribute used to prefix connection detail keys for sensitive
+	// Terraform attributes. We need this prefix to ensure that they are not
+	// overridden by any custom connection key configured which would break
+	// our ability to build tfstate back.
+	PrefixAttribute = "attribute."
```

Comment:
Given that now we return error if user tried to override an attribute key, do we still need this?